content/webapp/pages/event.js

```diff
@@ -316,9 +316,18 @@ class EventPage extends Component<Props, State> {
             }
 
             {!event.eventbriteId && !event.bookingEnquiryTeam && !(event.schedule && event.schedule.length > 1) &&
-              <div className={spacing({s: 4}, {margin: ['bottom']})}>
-                <Message text='Just turn up' />
-              </div>
+              <Fragment>
```

Comment:
Could this be simplified to `<Message text={event.hasEarlyRegistration ? 'Arrive early to register : 'Just turn up' />` ?

Reply:
Could have been, yip.